streams/src/test/java/org/apache/kafka/streams/integration/JoinIntegrationTest.java

```diff
@@ -147,7 +147,8 @@ public void shouldCountClicksPerRegion() throws Exception {
         // StreamsConfig configuration (so we can retrieve whatever state directory Streams came up
         // with automatically) we don't need to set this anymore and can update `purgeLocalStreamsState`
         // accordingly.
-        streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, "/tmp/kafka-streams");
+        streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG,
+                                 "/tmp/kstreams-join-integration-test");
```

Comment:
Wouldn't be better to use random dirs like Kafka normally does instead of manually setting the directory name in each test?